### ISSUE TITLE
Rustdoc should give an error if you set both `doc(inline)` and `doc(no_inline)`

### ISSUE THREAD

**Issue by jyn514** [rust-lang/rust#80275]
Given the following code:
```rust
#![crate_type = "lib"]

pub mod inner {
    pub struct S;
}

#[doc(inline)]
#[doc(no_inline)]
pub use inner::S;
```

Rustdoc will generate ![image](https://user-images.githubusercontent.com/23638587/102828782-8c7ad800-43b3-11eb-83d1-7745e149a6f1.png)
This is bogus: rustdoc is ignoring one of the attributes. Instead, it should give an error that you can't apply both attributes to the same item.

Thought of while reviewing https://github.com/rust-lang/rust/pull/80267.

**Comment by LeSeulArtichaut** [comment]
I'll take a look at this as a first rustdoc issue :)
